src/themes/default/pages/Product.vue

```diff
@@ -350,17 +352,24 @@ export default {
     getInputName () {
       if (this.isSimpleOrConfigurable && !this.isProductLoading) { return this.$i18n.t('Quantity available', { qty: this.quantity }) }
       return this.$i18n.t('Quantity')
+    },
+    isAddToCartDisabled () {
+      return (!this.$v.getCurrentProduct.qty.minValue || !this.$v.getCurrentProduct.qty.maxValue || !this.$v.getCurrentProduct.qty.numeric) || (!this.quantity && this.isSimpleOrConfigurable && !this.isProductLoading)
```

Comment:
maybe use here few regular `if` statements because this one inline if is a bit hard to read.

Reply:
sure, I just moved it here from Html, but I think it could also be something like
```suggestion
      return this.$v.$invalid || (!this.quantity && this.isSimpleOrConfigurable && !this.isProductLoading)
```
I'll short it

Reply:
done :)